truffleruby/src/main/ruby/post-boot/shims.rb

```diff
@@ -13,4 +13,12 @@ def self.default_dir
       File.expand_path(File.join(ConfigMap[:libdir], 'ruby', 'gems', 'shared'))
     end
   end
+
+  begin
+    gem 'did_you_mean'
+    require 'did_you_mean'
+  rescue Gem::LoadError, LoadError => e
+  end # if defined?(DidYouMean) # TODO BJF 29-Jan-2017 The definition of this constant is used to
+                                # control the ruby option to disable did_you_mean
```

Comment:
Argh. How about making a single consolidated `post-boot/post-boot.rb` then that integrate both files?